b2share/modules/schemas/alembic/226892354af5_create_schemas_branch.py

```diff
@@ -0,0 +1,24 @@
+"""Create schemas branch.
+
+Revision ID: 226892354af5
+Revises:
+Create Date: 2017-04-25 18:03:12.388159
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '226892354af5'
+down_revision = 'dbdbc1b19cf2'
+branch_labels = (u'b2share_schemas',)
+depends_on = 'dbdbc1b19cf2'
```

Comment:
as mentioned above, a comment pointing to the module containing this alembic script would be nice.